README.md

```diff
@@ -239,18 +239,23 @@ if (isset($response['errors'])) {
 }
 ```
 
-## Unit tests
+## Contributing
+
+### Setting up your environment (automatically)
+In order to get started quickly, we recommend to use [DDEV](https://ddev.readthedocs.io/en/stable/) which sets up things automatically for you. It clones [https://github.com/mautic/mautic](mautic/mautic), sets up a local instance for you, and connects the API library tests to that instance.
+
+Just run `ddev start` to get started! Our first-run experience will guide you through.
+
+### Setting up your environment (manually)
+If you know what you're doing and want to set up your local environment manually, just make sure to copy `/tests/local.config.php.dist` to `/tests/local.config.php`, and fill in the required settings. To make things easy, make sure to use Basic Authentication as the authentication method.
```

Comment:
Let's try to avoid competence-based assumptions:

```suggestion
If you want to set up your local environment manually, ensure that you copy `/tests/local.config.php.dist` to `/tests/local.config.php`, and fill in the required settings. We recommend using the Basic Authentication method to get up and running quickly.
```